lib/matplotlib/tests/test_backend_webagg.py

```diff
@@ -23,9 +24,9 @@ def test_webagg_fallback(backend):
         + "print(plt.get_backend());"
         f"assert '{backend}' == plt.get_backend().lower();"
     )
-    ret = subprocess.call([sys.executable, "-c", test_code], env=env)
-
-    assert ret == 0
+    subprocess_run_for_testing(
+        [sys.executable, "-c", test_code], env=env, check=True
+    )
```

Comment:
This fits in a line?
```suggestion
    subprocess_run_for_testing([sys.executable, "-c", test_code], env=env, check=True)
```